src/CMakeLists.txt

```diff
@@ -0,0 +1,102 @@
+add_library(pplib_utils STATIC
+    util/utilcrypt.c
+    util/utiliof.c
+    util/utillog.c
+    util/utilmd5.c
+    util/utilmem.c
+    util/utilnumber.c
+    util/utilsha.c
```

Comment:
* I can try to fix his `Makefile` later.
* What do you mean by “have the header file on the list to show it”?  The arguments to `add_library` should only list translation units.  Headers are not compiled.

Reply:
I know it is technically not required to list them in order to compile the project, but if you use VS, they simply don't show up, so now I'm always consistently adding the headers to my CMake projects, else it's tedious to do the development (you don't see the headers, so that you could click on them and edit them easily). https://stackoverflow.com/questions/13703647/how-to-properly-add-include-directories-with-cmake

Reply:
Would be perect if you leave the Makefile intact, but if for any reason Makefile must be used in CMake build flow, then I can use Makefile.pure or pj.mk or whatever suits.

Reply:
To be clear, I have nothing against CMake, I just don't need it and don't use it :) I've got used to several building scripts / toolchains in the same project; eg. usually I have at least a Makefile (for make) and Makefile.vc (for nmake). Could you [please explain why does CMake stuff conflicts with Makefile?

Reply:
It doesn't conflict at all.  I just moved files with broke the `Makefile`, but that is fixed in 
d5bd839.